src/ui/triggers_main_area.ui

```diff
@@ -35,7 +35,16 @@
    <property name="spacing">
     <number>0</number>
    </property>
-   <property name="margin">
+   <property name="leftMargin">
```

Comment:
'Tis a pity that although the new `uic` is perfectly capable of handling the simple case of where all the margins are all the same and specified as "margin", the later versions of the Qt Designer plug-in insist on splitting it and storing four identical values for the four aspects separately...

_I guess there is not much that can be done but to grit our teeth and accept the **git noise** this induces - or by taking action (hand-editing the file afterwards to revert to previous form) to end it_ and not everyone has the patience or sheer bloody-mindedness to do _that_ 😜